scripts/test_all.sh

```diff
@@ -44,7 +44,7 @@ echo_and_run bazel run //internal/node/test:has_deps_hybrid
 echo_and_run bazel run //internal/e2e/fine_grained_no_bin:index
 echo_and_run bazel run @fine_grained_deps_yarn//typescript/bin:tsc
 
-# bazel test @program_example//... # DOES NOT WORK WITH --nolegacy_external_runfiles
+# bazel test @examples_program//... # DOES NOT WORK WITH --nolegacy_external_runfiles
```

Comment:
generally I don't think this rename is worth all the churn in the repo

what's wrong with foo_example living in examples/foo ? seems intuitive to me

Reply:
I prefer the workspace name matching the path order. Makes it easier to trace when you see:

```
      # We should also be able to test targets in a different workspace
      # TODO(gmagolan): move these tests into `build` job once
      # https://github.com/bazelbuild/bazel/issues/6481 is resolved
      - run: bazel test @examples_program//...
      - run: bazel test @packages_example//...
```

But don't feel strongly either way. The churn is only in 3 locations.